core/src/main/scala/kafka/server/AdminManager.scala

```diff
@@ -607,4 +610,17 @@ class AdminManager(val config: KafkaConfig,
     val readOnly = !allNames.exists(DynamicBrokerConfig.AllDynamicConfigs.contains)
     new DescribeConfigsResponse.ConfigEntry(name, valueAsString, source, isSensitive, readOnly, synonyms.asJava)
   }
+
+  private def resolveVariableConfigs(propsOriginal: Properties): Properties = {
+    val props = new Properties
+    propsOriginal.put("bootstrap.servers", "localhost:1")
+    val config = new AdminClientConfig(propsOriginal)
```

Comment:
Given that we don't use `AdminClientConfig`'s overridden `postProcessParsedConfig()` method, does it make sense to use `AbstractConfig` here?

Reply:
Updated code using AbstractConfig.